developer-docs-site/docs/guides/move-guides/index.md

```diff
@@ -25,6 +25,7 @@ Start here to learn how the Move language works on the Aptos blockchain.
 - ### [Mint NFT with Aptos CLI](./mint-nft-cli.md)
 - ### [Upgrading Move Code](upgrading-move-code.md)
 - ### [Aptos Move Examples](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples)
+- ### [Move language Discord](https://discord.com/channels/945856774056083548/955573698868432896)
```

Comment:
This doesn't look like the right kind of link for a Discord invite.

I think in general we've been pointing people at the Aptos labs discord for move help, since there are nuances that are specific to the deployment of Move

Reply:
> This doesn't look like the right kind of link for a Discord invite.
> 
> I think in general we've been pointing people at the Aptos labs discord for move help, since there are nuances that are specific to the deployment of Move

Yes, this goes directly to the #move-language channel in Aptos Labs Discord. We already link to the main Discord invitation link prominently from the home page in a few places:
https://aptos.dev/

If we don't want to link to the channel, I can remove this entry.

Reply:
Yeah, let's... link directly to the Discord server rather than the channel.  Or better, just have 2 links, one for the server, and one for the channel